src/audio.rs

```diff
@@ -0,0 +1,86 @@
+use crate::twilio_response;
+
+pub struct BufferData {
+    pub inbound_buffer: Vec<u8>,
+    pub outbound_buffer: Vec<u8>,
+    pub inbound_last_timestamp: u32,
+    pub outbound_last_timestamp: u32,
+}
+
+/// if more than 400 ms of audio is available in both inbound and outbound
+/// audio buffers, we will drain that audio data, mix it together as correct
+/// stereo audio, taking into account any potentially dropped audio packets,
+/// and return the mixed audio
+pub fn process_twilio_media(
+    media: twilio_response::EventMedia,
+    mut buffer_data: &mut BufferData,
+) -> Option<Vec<u8>> {
+    // NOTE: when Twilio sends media data, it should send 20 ms audio chunks at a time, where each ms of audio is 8 bytes
```

Comment:
Wrapping this comment at the 100 character mark
```suggestion
    // NOTE: when Twilio sends media data, it should send 20 ms audio chunks at a time, where each
    // ms of audio is 8 bytes
```